src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/silence-form/silence-form.component.html

```diff
@@ -1,218 +1,194 @@
-          </label>
-          <div class="cd-col-form-input">
-            <input class="form-control"
-                   formControlName="startsAt"
-                   [ngbPopover]="popStart"
-                   triggers="manual"
-                   #ps="ngbPopover"
-                   (click)="ps.open()"
-                   (keypress)="ps.close()">
-            <span *ngIf="form.showError('startsAt', formDir, 'required')"
-                  class="invalid-feedback"
-                  i18n>This field is required!</span>
-          </div>
-        </div>
-
-        <!-- Duration -->
-        <div class="form-group row">
-          <label class="cd-col-form-label required"
-                 for="duration"
-                 i18n>Duration</label>
-          <div class="cd-col-form-input">
-            <input class="form-control"
-                   formControlName="duration"
-                   id="duration"
-                   name="duration"
-                   type="text">
-            <span *ngIf="form.showError('duration', formDir, 'required')"
-                  class="invalid-feedback"
-                  i18n>This field is required!</span>
-          </div>
-        </div>
-
-        <!-- End time -->
-        <div class="form-group row">
-          <label class="cd-col-form-label required"
-                 for="ends-at"
-                 i18n>End time</label>
-          <div class="cd-col-form-input">
-            <input class="form-control"
-                   formControlName="endsAt"
-                   [ngbPopover]="popEnd"
-                   triggers="manual"
-                   #pe="ngbPopover"
-                   (click)="pe.open()"
-                   (keypress)="pe.close()">
-            <span *ngIf="form.showError('endsAt', formDir, 'required')"
-                  class="invalid-feedback"
-                  i18n>This field is required!</span>
-          </div>
+  <div class="form-header">
+    {{ action | titlecase }} {{ resource | upperFirst }}
+  </div>
+  <div class="form-item form-item-append"
+       cdsRow>
+    <!--Creator -->
+    <div cdsCol>
+      <cds-text-label labelInputID="CreatedBy"
+                      i18n
+                      i18n-helperText
+                      cdRequiredField="CreatedBy"
+                      [invalid]="silenceForm.controls.createdBy.invalid && silenceForm.controls.createdBy.dirty"
+                      [invalidText]="createdByError">Created By
+      <input cdsText
+             type="text"
+             id="createdBy"
+             formControlName="createdBy"
+             [invalid]="silenceForm.controls.createdBy.invalid && silenceForm.controls.createdBy.dirty"/>
+      </cds-text-label>
+      <ng-template #createdByError>
+        <span class="invalid-feedback"
+              *ngIf="silenceForm.showError('createdBy', formDir, 'required')"
+              i18n>This field is required.</span>
+      </ng-template>
+    </div>
+    <!-- Comment -->
```

Comment:
please add and extra empty line before each items as it was before. that makes the code easier to read especially with bigger forms.